packages/terra-tabs/tests/wdio/tabs-spec.js

```diff
@@ -15,26 +15,26 @@ describe('Tabs - Responsive', () => {
         browser.setViewportSize(viewport);
         browser.moveToObject('.tabContent');
       });
-      Terra.should.matchScreenshot({ viewports });
-      Terra.should.beAccessible({ viewports });
+      Terra.should.matchScreenshot();
+      Terra.should.beAccessible({ viewports: [viewport] });
```

Comment:
Actually this an be updated to be `Terra.should.beAccessible();`
By not passing a viewport, it uses the current viewport size (which is set in the loop above).

Reply:
Interesting, we should get this updated before we merge.